sql/CMakeLists.txt

```diff
@@ -1,21 +1,7 @@
-set(EXT_SQL_FILE ${PROJECT_NAME}--${PROJECT_VERSION_MOD}.sql)
-set(EXT_SQL_UPDATE_FILE ${PROJECT_NAME}--${UPDATE_FROM_VERSION}--${PROJECT_VERSION_MOD}.sql)
-set(EXT_SQL_UPDATE_PRE_FILE updates/pre-${UPDATE_FROM_VERSION}--${PROJECT_VERSION_MOD}.sql)
-set(EXT_SQL_UPDATE_POST_FILE updates/post-${UPDATE_FROM_VERSION}--${PROJECT_VERSION_MOD}.sql)
+set(INSTALL_FILE ${PROJECT_NAME}--${PROJECT_VERSION_MOD}.sql)
 
-#These files called before anything else
-set(PRE_UPDATE_FILES
-  updates/pre-global.sql
-  #trigger functions need to go first to associate them with new .so before they are called
-  trigger_functions.sql
-)
-
-set(PRE_INSTALL_FILES
-  schemas.sql
-  trigger_functions.sql
-)
-
-set(COMMON_FILES
+set(SOURCE_FILES
+  schemas.sql # Must be first
   tables.sql
```

Comment:
I guess we could do that, but it doesn't hurt to include `tables.sql` in its current form (clearer semantics). Besides, you only need to declare new tables in update scripts before `tables.sql` if you need them for your update. We could also add a post script if table manipulations are needed after `tables.sql`

Reply:
So the scenario I am worried about is: I add a table in version 2 and then I want to alter it in version 3 (for example: add a column). Now if the table isn't in the modfile for version 2, I need to add it in the modfile for version 3 before I do the ALTER, which is weird since I actually added it in version 2. This is needed for the jumpfile 1--3. I think it's much more clear if the CREATE TABLE is in modfile 2.

Reply:
I also think the idea that only function definition are in the "install suffix" that goes after all the modfiles makes this easier to reason about. Then, everything else goes into modfiles. It's clean and clear semantics. Besides, most table changes (except for CREATE) already need to go into modfiles so this isn't a big change --- but one that makes the code more understandable IMHO.

Reply:
Maybe a solution is to always include `tables.sql` and `schema.sql` before the modfiles?